arlas-tests/src/test/java/io/arlas/server/tests/ogc/csw/CSWServiceIT.java

```diff
@@ -115,7 +115,7 @@ public void testGetRecordByIdWithCollectionBasedColumnFilter() throws Exception
         String id = getRecordId(collectionName);
 
         handleGetRecordById(
-                get("GetRecordById", Arrays.asList(new ImmutablePair<>("ID", id)), Optional.of(" ")),
+                get("GetRecordById", Arrays.asList(new ImmutablePair<>("ID", id)), Optional.empty()),
                 collectionName);
         handleGetRecordById(
                 get("GetRecordById", Arrays.asList(new ImmutablePair<>("ID", id)), Optional.of("panda:anyField")),
```

Comment:
Shouldn't we add a `handleUnavailableCollection` test as it is the case for the exploration endpoints tests ?

Reply:
Ok